ramls/loan.json

```diff
@@ -80,6 +80,10 @@
       "pattern": "^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}$",
       "type": "string"
     },
+    "patronGroupIdAtCheckout": {
```

Comment:
Changes in *.json require minor version change in ModuleDescriptor and raml files